inventory/src/main/java/org/flyve/inventory/categories/Cameras.java

```diff
@@ -445,19 +445,20 @@ public String getModel(int index) {
      * Get support camera
      * @return String support camera
      */
-    public String getSupportValue() {
+    public ArrayList<String> getSupportValue() {
         ArrayList<String> arrayList = new ArrayList<>();
         Iterator it = getCatInfoCamera("/system/lib/libcameracustom.so", "SENSOR_DRVNAME_", 100).iterator();
         while (it.hasNext()) {
             arrayList.add(((String) it.next()).toLowerCase(Locale.US));
         }
-        StringBuilder values = new StringBuilder();
+        return arrayList;
```

Comment:
This method could return an empty array validate if the value is empty the tag should not appear on the xml or json